docs/pyproject.md

```diff
@@ -106,14 +106,28 @@ maintainers = [
 
 ## readme
 
-The readme file of the package. **Optional**
+The path, or array of paths, to the readme file(s) of the package. **Optional**
 
-The file can be either `README.rst` or `README.md`.
+The file should be in either Markdown (.md) or reStructuredText (.rst)
```

Comment:
As #5357 is already present, this is no longer true. Somehow adding this to docs here was missed. Could you add this, please?

Reply:
oh nice, yes I will add that

Reply:
okay that's added now